big_tests/tests/graphql_helper.erl

```diff
@@ -29,10 +29,16 @@ execute_command(Category, Command, Args, Config) ->
     Protocol = ?config(protocol, Config),
     execute_command(Category, Command, Args, Config, Protocol).
 
+execute_domain_admin_command(Category, Command, Args, Config) ->
+    execute_command(Category, Command, Args, Config, domain).
+
 %% Admin commands can be executed as GraphQL over HTTP or with CLI (mongooseimctl)
 execute_command(Category, Command, Args, Config, http) ->
     #{Category := #{Command := #{doc := Doc}}} = get_specs(),
     execute_auth(#{query => Doc, variables => Args}, Config);
+execute_command(Category, Command, Args, Config, domain) ->
```

Comment:
Please don't use `domain` for Protocol. The protocol is still HTTP. Just put lines 40-41 directly in the body of `execute_domain_admin_command/4`